Simplenote/NoticeFactory.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+struct NoticeFactory {
+    static func linkCopied() -> Notice {
+        return Notice(message: Messages.copyLink, action: nil)
+    }
+
+    static func publishing() -> Notice {
+        Notice(message: Messages.publishing, action: nil)
+    }
+
+    static func unpublishing() -> Notice {
+        return Notice(message: Messages.unpublishing, action: nil)
```

Comment:
Nipticky: Tiny inconsistency between this and L9 (good to drop return if it's the only statement!)